tools/depends/configure.ac

```diff
@@ -380,7 +385,7 @@ case $host in
     platform_cxxflags="$platform_cflags"
     platform_os="linux"
     meson_system="linux"
-    target_platform="wayland gbm"
+    target_platform="wayland"
```

Comment:
why? if gbm conflicts with webos, then you need to turn this off for webos only

Reply:
This is actually not true. There is gbm support on WebOS but the preferred way is wayland